rivertype/river_type.go

```diff
@@ -13,6 +13,10 @@ import (
 // return this error.
 var ErrNotFound = errors.New("not found")
 
+// ErrJobRunning is returned when a job is attempted to be deleted while it's
+// running.
+var ErrJobRunning = errors.New("job is running")
```

Comment:
What do you think about a slightly more descriptive error message here in case this error bubbles up to an end user somewhere? Something like:

```suggestion
var ErrJobRunning = errors.New("running jobs cannot be deleted")
```